common.gypi

```diff
@@ -36,7 +36,7 @@
 
     # Reset this number to 0 on major V8 upgrades.
     # Increment by one for each non-official patch applied to deps/v8.
-    'v8_embedder_string': '-node.20',
+    'v8_embedder_string': '-node.21',
```

Comment:
Comment above says this should be incremented for each non-official patch, so probably each commit in this PR should bump the version up by one.